core/handler/cowboy_routing.ex

```diff
@@ -28,19 +30,37 @@ defmodule AntikytheraCore.Handler.CowboyRouting do
       @total_error_count_route,
       @per_app_error_count_route,
     ]
-    [{:_, path_rules}]
+    case default_routing_gear() do
+      :error      -> [{:_, path_rules}]
+      {:ok, gear} -> [{:_, gear_routes(gear) ++ path_rules}]
```

Comment:
As far as I tested on the current master on iex,
* `http://gearname.localhost:8080` works
* `http://gearname.myhostname:8080` doesn't work
* `http://myhostname:8080/healthcheck` works even if I accessed from an other PC

It looks like I can access my gear from an other PC using `http://myhostname:8080` url after this change.
Is this correct?

I think it's a bit insecure if other people can access my developing gear.

Can we limit this to `*.localhost` or `*.BASE_DOMAIN`?

Reply:
Thanks. I'll fix it.

Reply:
fixed. 7ac54ff7808f60fdd148c1903a7aaa07cd6eaf9d